### PR TITLE
[ASDisplayNode] Add allowsGroupOpacity property

### PR DESCRIPTION
**Pull request by levi**
Since we're disabling it in our app, provide easy access for changing the layer's behavior.

### CODE REVIEW

File: AsyncDisplayKitTests/ASDisplayNodeTests.m [hunk 1]
```diff
@@ -400,6 +401,7 @@ - (void)checkValuesMatchSetValues:(ASDisplayNode *)node isLayerBacked:(BOOL)isLa
   XCTAssertEqual(YES, node.clipsToBounds, @"clipsToBounds broken %@", hasLoadedView);
   XCTAssertEqual(NO, node.opaque, @"opaque broken %@", hasLoadedView);
   XCTAssertEqual(YES, node.needsDisplayOnBoundsChange, @"needsDisplayOnBoundsChange broken %@", hasLoadedView);
+  XCTAssertEqual(NO, node.allowsGroupOpacity, @"allowsEdgeAntialiasing broken %@", hasLoadedView);
```
Comment (Adlai-Holler) [inline AsyncDisplayKitTests/ASDisplayNodeTests.m]: Update message

**Comment by Adlai-Holler** [comment]
Looks like the CI is failing because it wants you to add accessors into `ASDisplayNode+UIViewBridge.mm` as well

**Comment by levi** [comment]
@Adlai-Holler let's see if this fixes it.

File: AsyncDisplayKitTests/ASDisplayNodeTests.m (hunk 1 above)
Comment (appleguy) [inline AsyncDisplayKitTests/ASDisplayNodeTests.m]: @levi Thanks for adding this new bridged property, it's great to have!  Indeed these two assertions both need their string updated

**Comment by appleguy** [comment]
@levi interesting, this could be a true failure.

16:23:30 ASDisplayNodeTests
16:23:30   testDefaultPropertyValuesLayer, ((YES) equal to (node.allowsGroupOpacity)) failed: ("<01>") is not equal to ("<00>") - default allowsEdgeAntialiasing broken after loading layer
16:23:30   /Users/engineering/jenkins_root/workspace/AsyncDisplayKit-PRs/AsyncDisplayKitTests/ASDisplayNodeTests.m:306
16:23:30   ```
16:23:30   XCTAssertEqual(NO, node.needsDisplayOnBoundsChange, @"default needsDisplayOnBoundsChange broken %@", hasLoadedView);
16:23:30   XCTAssertEqual(YES, node.allowsGroupOpacity, @"default allowsEdgeAntialiasing broken %@", hasLoadedView);
16:23:30   XCTAssertEqual(NO, node.allowsEdgeAntialiasing, @"default allowsEdgeAntialiasing broken %@", hasLoadedView);
16:23:30   ```
16:23:30 
16:23:30   testDefaultPropertyValuesView, ((YES) equal to (node.allowsGroupOpacity)) failed: ("<01>") is not equal to ("<00>") - default allowsEdgeAntialiasing broken after loading view
16:23:30   /Users/engineering/jenkins_root/workspace/AsyncDisplayKit-PRs/AsyncDisplayKitTests/ASDisplayNodeTests.m:306
16:23:30   ```
16:23:30   XCTAssertEqual(NO, node.needsDisplayOnBoundsChange, @"default needsDisplayOnBoundsChange broken %@", hasLoadedView);
16:23:30   XCTAssertEqual(YES, node.allowsGroupOpacity, @"default allowsEdgeAntialiasing broken %@", hasLoadedView);
16:23:30   XCTAssertEqual(NO, node.allowsEdgeAntialiasing, @"default allowsEdgeAntialiasing broken %@", hasLoadedView);
16:23:30   ```
16:23:30 
16:23:30 
16:23:30 	 Executed 358 tests, with 2 failures (0 unexpected) in 8.897 (9.125) seconds


@levi note that the DEFAULT for this property will depend on the Info.plist, so maybe we shouldn't verify the default value.

Full build log: http://jenkinsmaster-vpc-007.ec2.pin220.com/job/AsyncDisplayKit-PRs/491/console

**Comment by levi** [comment]
Yeah, it's a true failure. I haven't built it locally and was hoping the CI environment would do it for me :) Will see if I can clean this up tomorrow.

**Comment by maicki** [comment]
Hey @levi, you think you can get the problem solved soon?

**Comment by levi** [comment]
Not today, can you pull it down and fix it?

Cheers,
levi

**Comment by maicki** [comment]
Will do. Thanks.

**Comment by maicki** [comment]
Closing as it is replaced by: #2320

**Pull request closed without merging**